explorer/LICENSE

```diff
@@ -1,20 +0,0 @@
-Copyright (c) 2016 IOHK
```

Comment:
We have a `LICENSE` file in all directories with different packages (`tools`, `update`, etc.). I think it should be in `explorer` too.